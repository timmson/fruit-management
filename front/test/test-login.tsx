import React from "react"
import renderer from "react-test-renderer"
import Login from "../src/login";

describe("Login should", () => {

    test("return login form", () => {
        const component = renderer.create(<Login/>)

        expect(component.toJSON()).toMatchSnapshot()

        component.unmount()
    })

})