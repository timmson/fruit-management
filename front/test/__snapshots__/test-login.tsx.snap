// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login should return login form 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col text-center"
    >
      <img
        alt="Fruit Management"
        src="img/logo.gif"
      />
    </div>
  </div>
  <div
    className="row mt-4"
  >
    <div
      className="col text-center"
    >
      <h1>
        Fruit Management
      </h1>
      <p
        style={
          {
            "color": "#FF0000",
          }
        }
      />
    </div>
  </div>
  <form
    id="loginForm"
    onSubmit={[Function]}
  >
    <div
      className="row mt-4"
    >
      <div
        className="col-5 text-end"
      >
        <label
          className="col-form-label"
          htmlFor="login"
        >
          Логин:
        </label>
      </div>
      <div
        className="col-2 text-start"
      >
        <input
          className="form-control"
          id="login"
          name="login"
          onChange={[Function]}
          placeholder="fruit"
          type="text"
          value=""
        />
      </div>
      <div
        className="col-5 text-end"
      >
         
      </div>
    </div>
    <div
      className="row mt-2"
    >
      <div
        className="col-5 text-end"
      >
        <label
          className="col-form-label"
          htmlFor="pass"
        >
          Пароль:
        </label>
      </div>
      <div
        className="col-2 text-start"
      >
        <input
          className="form-control"
          id="pass"
          name="pass"
          onChange={[Function]}
          placeholder="*****"
          type="password"
          value=""
        />
      </div>
      <div
        className="col-5 text-end"
      >
         
      </div>
    </div>
    <div
      className="row mt-3"
    >
      <div
        className="col text-center"
      >
        <input
          className="btn btn-primary corp"
          id="sub"
          name="sub"
          type="submit"
          value="Войти"
        />
      </div>
    </div>
  </form>
  <div
    className="row mt-4"
  >
    <div
      className="col-3"
    >
       
    </div>
    <div
      className="col-1"
    >
      <img
        alt="powered by xhtml"
        src="img/xhtml.png"
      />
    </div>
    <div
      className="col-1"
    >
      <img
        alt="powered by css"
        src="img/css.png"
      />
    </div>
    <div
      className="col-1"
    >
      <img
        alt="powered by php"
        src="img/php.png"
      />
    </div>
    <div
      className="col-1"
    >
      <img
        alt="powered by smarty"
        src="img/smarty.png"
      />
    </div>
    <div
      className="col-1"
    >
      <img
        alt="powered by ajax"
        src="img/ajax.png"
      />
    </div>
    <div
      className="col-1"
    >
      <img
        alt="powered by natali"
        src="img/natali.png"
      />
    </div>
    <div
      className="col-3"
    >
       
    </div>
  </div>
</div>
`;
